### PR TITLE
Add render to CreateCluster

### PR DESCRIPTION
**Pull request by zheng1**

### CODE REVIEW

File: pkg/plugins/qingcloud/parser_test.go [hunk 1]
```diff
@@ -691,7 +710,7 @@ func TestParse(t *testing.T) {
 		if hbaseClusterNodes[index] != *clusterWrapper.ClusterNodes[index] {
 			t.Errorf("Cluster node [%s] not equal.", index)
 			t.Logf("ori: %s", hbaseClusterNodes[index].Role)
```
Comment (chilianyi) [inline pkg/plugins/qingcloud/parser_test.go]: need +v?

File: pkg/plugins/qingcloud/provider.go [hunk 2]
```diff
@@ -35,17 +40,44 @@ func (p *Provider) ParseClusterConf(versionId, conf string) (*models.ClusterWrap
...
+		resp, err := appManagerClient.GetAppVersionPackage(ctx, req)
 		if err != nil {
 			logger.Errorf("Get app version [%s] package failed: %v", versionId, err)
 			return nil, err
 		}
 
-		// TODO after rendered, got the final conf
+		appPackage, err := devkit.LoadArchive(bytes.NewReader(resp.GetPackage()))
+		if err != nil {
+			logger.Errorf("Load app version [%s] package failed: %v", versionId, err)
```
Comment (chilianyi) [inline pkg/plugins/qingcloud/provider.go]: +v?

File: pkg/plugins/qingcloud/provider.go [hunk 3]
```diff
@@ -22,6 +26,7 @@ type Provider struct {
 }
 
 func (p *Provider) ParseClusterConf(versionId, conf string) (*models.ClusterWrapper, error) {
+	cluster := app.Cluster{}
```
Comment (chilianyi) [inline pkg/plugins/qingcloud/provider.go]: Can app.Cluster be changed to other name, like clusterTmpl or clusterJsonTmpl. 
Cluster is easy to be misunderstand.

File: pkg/plugins/qingcloud/provider_test.go [hunk 4]
```diff
@@ -43,22 +43,18 @@ func testCreateCluster(t *testing.T, frame *vmbased.Frame) {
...
 
 	if len(result) != len(expectResult) {
-		t.Errorf("Expect [%d] task layer, while get [%s] task layer", len(expectResult), len(result))
+		t.Errorf("Expect [%d] task layer, while get [%d] task layer", len(expectResult), len(result))
 	}
 
 	for index := range result {
 		if result[index] != expectResult[index] {
-			t.Errorf("Index [%d] expect [%s], while get [%s]", index, expectResult[index], result[index])
+			t.Errorf("Index [%d] expect [%v], while get [%v]", index, expectResult[index], result[index])
```
Comment (chilianyi) [inline pkg/plugins/qingcloud/provider_test.go]: +v?

File: pkg/plugins/qingcloud/provider.go (hunk 3 above)
Comment (zheng1) [reply pkg/plugins/qingcloud/provider.go]: done

**Pull request merged**